### ISSUE TITLE
Failed to connect to Discord

### ISSUE THREAD

**Issue by Ephemeralic** [demaisj/soundcloud-rp#17]
Everything seems right. Running the service with the DEBUG parameter applied shows that it is able to grab the correct track, but cannot find my Discord process.
![image](https://user-images.githubusercontent.com/40898962/48171950-fc10d480-e2cb-11e8-822c-17dcc6df8dde.png)

**Comment by demaisj** [comment]
Hi,
I don't really know where this problem could come from.

Have you tried restarting the discord client ?
Is the discord client running on your pc ?
What os/distribution are you using ?
What node version are you using ?

**Comment by Ephemeralic** [comment]
Yes, I have tried restarting the Discord client with Ctrl+R. No success.
The Discord client is running on the same PC that SoundcloudRPC is running on.
I am using Ubuntu 18.10, Node version 10.13.0.
